modules/ROOT/pages/associate-environments.adoc

```diff
@@ -5,15 +5,60 @@ endif::[]
 
 Before deploying applications or API gateways to Anypoint Runtime Fabric, you must associate a Runtime Fabric with one or more Anypoint Platform environments. When deploying a Mule application, you cannot select a Runtime Fabric as a deployment target until you have performed this step in the environment where you want to deploy.
 
+== Associate Business Groups
+
+You can share Runtime Fabric with an organization's business groups. This allows other business groups to deploy applications to a single Runtime Fabric. To use this feature, the business groups must fall under the scope of the organization the Runtime Fabric is registered to.
+
+To associate a Runtime Fabric across business groups:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select the *Runtime Fabric* tab on the left sidebar.
+. Select the Runtime Fabric to manage.
+. On the *Environments* tab, click *Add business group*.
+. Select the business groups from the drop-down list:
+
+** *This business group and its children*
++ 
+This option allows you to select either all environments, all sandbox environment or all production environments for the selected business group and all its children.
+
+** *Selected business groups* 
++
+Enter the business group name in the *Add business group* field and select it from the list. This option allows you to select specific environments for specific child business groups.
+
+[start=6]
+. Click *Save Changes* or *Discard Changes* to discard or confirm the association.
+
+
+
+== Associate Environments
+
 To associate an environment to a Runtime Fabric:
 
 . From Anypoint Platform, select *Runtime Manager*.
 . Select the *Runtime Fabric* section on the left sidebar.
 . Select the Runtime Fabric based on the name used during installation.
-. From the *Associated Environments* tab, select the environment you want to associate with this Runtime Fabric.
-. Click *Apply* to confirm the changes.
+. Click the *Environments* tab.
+. Select the environments from the drop-down list:
 
-== Production Environments
+** *All Environment* +
+Associates the private space with all sandbox and production environments in the business group and all its children, or in specific child business organizations.
```

Comment:
It should be `Associates the Runtime Fabric` for this all all subsequent statements.